sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -128,6 +128,12 @@
 
   /* Global settings */
   String SONAR_HOME = "SONAR_HOME";
+
+  /**
+   * @deprecated since 6.7. This feature is deprecated in favor of the new branch feature.
+   * @see <a href="https://redirect.sonarsource.com/doc/branches.html">https://redirect.sonarsource.com/doc/branches.html/a>
+   */
+  @Deprecated
   String PROJECT_BRANCH_PROPERTY = "sonar.branch";
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)